Mage.Client/src/main/java/mage/client/draft/DraftPanel.java

```diff
@@ -304,7 +304,9 @@ public void loadBooster(DraftPickView draftPickView) {
              countdown.start();
          }
          
-         SessionHandler.setBoosterLoaded(draftId); // confirm to the server that the booster has been loaded
+         if (!draftBooster.isEmptyGrid()) {
+            SessionHandler.setBoosterLoaded(draftId); // confirm to the server that the booster has been successfully loaded
+         }
```

Comment:
What happens if this doesn't get called? Does the server keep sending boosters?

Reply:
Yes, and I expanded the comment to include that.